rules_ruby_gapic/docker_testing/Dockerfile

```diff
@@ -32,13 +32,14 @@ RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add - \
 
 ### Requirement: system has `find` to remove the empty-spaced files
 
-### Requirement: system has `git` to get the gapic repo
+### Requirement: system has `wget` to get the gapic-generator-ruby repo
+### Requirement: `tar` and `unzip` are both used to produce packaged built libraries
 ### Requirement: system has `make` to build ruby
 ### Requirement: system has `zlib` and `zlib-dev`, `libreadline` and `libreadline-dev`, `openssl` and `openssl-dev`, all needed for rubygems to work
 RUN apt-get update && apt-get install -y --no-install-recommends \
-        git \
         wget \
         tar \
+        unzip \
```

Comment:
Please strongly consider relying on https://github.com/googleapis/googleapis/blob/master/.kokoro/Dockerfile everywhere where you need a docker file, and remove the custom one.

Reply:
I'll consider that at the time I'll be moving the docker example to `bazel_examples`, a planned chore

Reply:
:+1: